blivet/python-blivet.spec

```diff
@@ -15,7 +11,7 @@ License: LGPLv2+
 Group: System Environment/Libraries
 %global realname blivet
 %global realversion %{version}%{?prerelease}
-Source0: http://github.com/rhinstaller/blivet/archive/%{realname}-%{realversion}.tar.gz
+Source0: %{realname}-%{realversion}.tar.gz
```

Comment:
I'd leave URL here. If nothing more, as an info how to get new upstream version (`spectool --get-files` do that). You already have `(notdir ...)` in `SOURCE_ARCHIVE_NAME`, so it should just work, no?